### PR TITLE
DOCSP-20585: versioned API renamed to stable API

### PR DESCRIPTION
**Pull request by ccho-mongodb**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-20585

### Snooty build log:
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=61f43fb9382d7fe4ee37b77e

### Docs staging link (requires sign-in on MongoDB Corp SSO):
[ToC](https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-20585-stable-api/#fundamentals
)
[What's New](https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-20585-stable-api/whats-new/#what-s-new-in-4.3)
[Fundamentals ToC](https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-20585-stable-api/fundamentals/)
[Stable API](https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-20585-stable-api/fundamentals/stable-api/) (linked Server Manual page not published yet)

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [ ] Are all the links working?
- [x] Are the staging and workerpool job links in the PR description updated?

### If your page documents a concept, does it meet the following criteria?

- [ ] Target the [Jasmin persona](https://drive.google.com/file/d/14FbBOLCVxwSP6M9BK4Nz1Ir9tzxT8_02/view)
- [ ] Target the [Lucas persona](https://drive.google.com/file/d/1J2vqJxo7ldv7OP_obA9Q-avf0o_ju4Lk/view)

### CODE REVIEW

File: source/fundamentals/stable-api.txt [hunk 1]
```diff
@@ -104,11 +105,12 @@ section, see the following API Documentation:
 - `MongoClient <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoClient.html>`__
 
 .. _versioned-api-options:
+.. _stable-api-options:
 
-Versioned API Options
----------------------
+{+stable-api+} API Options
```
Comment (zach-carr) [inline source/fundamentals/stable-api.txt:110]: [issue] double API
```suggestion
{+stable-api+} Options
```

**Review by zach-carr — APPROVED** [review]
LGTM with 1 fix

Can you delete source/includes/fundamentals/code-snippets/VersionedApiExample.java since you now have StableApiExample.java?

File: config/redirects [hunk 2]
```diff
@@ -4,3 +4,5 @@ define: versions v4.3 v4.4 master
 
 raw: ${prefix}/ -> ${base}/current/
 raw: ${prefix}/master -> ${base}/upcoming/
+
+raw: ${prefix}/${version}/fundamentals/versioned-api/ -> ${prefix}/${version}/fundamentals/stable-api/
```
Comment (biniona-mongodb) [inline config/redirects:8]: Comment: I think the `raw` redirect doesn't let you use a [versioned variable
](https://docs.mongodb.com/meta/redirects/#rules)

```suggestion
[*-master]: ${prefix}/${version}/fundamentals/versioned-api/ -> ${prefix}/${version}/fundamentals/stable-api/
```

Comment (ccho-mongodb) [reply config/redirects:8]: Didn't know that, thanks

**Pull request merged**